support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/config.jelly

```diff
@@ -40,20 +40,20 @@ THE SOFTWARE.
     </f:entry>
     <f:entry title="Parameters">
         <div id="params"/>
+        <script>
+            function loadParams() {
+                var div = $$('params');
+                new Ajax.Request('${descriptor.descriptorUrl}/parameters?job=' + encodeURIComponent($$('${jobFieldId}').value) + '&amp;context=' + encodeURIComponent('${descriptor.context}'), {
+                    method : 'get',
+                    onSuccess : function(x) {
+                        div.innerHTML = x.responseText;
+                        Behaviour.applySubtree(div);
+                    },
+                    onFailure : function(x) {
+                        div.innerHTML = "<b>ERROR</b>: Failed to load parameter definitions: " + x.statusText;
+                    }
+                });
+            }
+        </script>
     </f:entry>
-    <script>
-        function loadParams() {
-            var div = $$('params');
-            new Ajax.Request('${descriptor.descriptorUrl}/parameters?job=' + encodeURIComponent($$('${jobFieldId}').value) + '&amp;context=' + encodeURIComponent('${descriptor.context}'), {
-                method : 'get',
-                onSuccess : function(x) {
-                    div.innerHTML = x.responseText;
-                    Behaviour.applySubtree(div);
-                },
-                onFailure : function(x) {
-                    div.innerHTML = "<b>ERROR</b>: Failed to load parameter definitions: " + x.statusText;
-                }
-            });
-        }
-    </script>
```

Comment:
So the issue with this ... once this gets assembled, the content returned to the browser has the `<script>` tags in the middle of the config form `<table>` elements (between the `<tr>`s), causing HtmlUnit to balk on it.